plugins/org.jboss.tools.vpe.reddeer/src/org/jboss/tools/vpe/reddeer/preview/editor/VPVEditor.java

```diff
@@ -31,7 +32,7 @@
 import org.jboss.tools.jst.web.ui.internal.editor.jspeditor.PalettePageImpl;
 import org.jboss.tools.jst.web.ui.palette.PaletteAdapter;
 
-public class VPVEditor extends JSPMultiPageEditor{
+public class VPVEditor extends VisualEditor{
```

Comment:
I guess you are planning to put some logic into VisualEditor in the future, right?